src/store/index.js

```diff
@@ -243,6 +249,53 @@ const store = new Vuex.Store({
         loopIndex++;
       })
       commit(mutations.SET_ACCOUNT_CARDS, tokenDetailsArrayProcessed);
+    },
+    [actions.LOAD_DEEP_URL_CARD]: async function ({
+      commit,
+      dispatch,
+      state
+    }, {
+      tokenId
+    }) {
+      if (state.deepUrlCardNumber === null) {
+        commit(mutations.SET_DEEP_URL_CARD_NUMBER, tokenId)
+
+      } else {
+        if (state.contract) {
+          const contract = await state.contract.deployed();
+          let accountToken = await contract.tokenDetails(tokenId);
```

Comment:
Would be nice to wrap this logic in a mapping method that returns an object in the right state/types etc.

Reply:
I agree. I think we can move this to the next iteration though as this works fine for now? Lets release then can do a big refactor. there are a number of other places that require this kind of refactor within the store.